packages/web/src/store/token-dashboard/sagas.ts

```diff
@@ -97,14 +98,22 @@ function* send() {
   })
 
   if (error) {
-    const errorState: ModalState = {
-      stage: 'SEND',
-      flowState: {
-        stage: 'ERROR',
-        error: error.payload.error ?? ''
+    if (error.payload.error === 'Missing social proof') {
+      yield all([
```

Comment:
yield all for first two and not third? checking if im missing some context there, just ensures they finish first?